pipeline_dp/aggregate_params.py

```diff
@@ -108,6 +116,7 @@ class AggregateParams:
     vector_max_norm: Optional[float] = None
     vector_size: Optional[int] = None
     contribution_bounds_already_enforced: bool = False
+    partition_selection_strategy: PartitionSelectionStrategy = PartitionSelectionStrategy.TRUNCATED_GEOMETRIC
```

Comment:
I think it makes sense to do the optimization in Go here:
if max_partitions_contributed > 3
partition_selection_strategy = GAUSSIAN_THRESHOLDING

(also below for select_partitions)

Reply:
I'm thinking to make at some point `auto`, which would do something like that. But to make it later, when PLD accounting is ready.

It can be more complicated if advanced composition is used, since for Gaussian_Thresholding advance composition is used automatically (just by computing sensitivity), but for TRUNCATED_GEOMETRIC not